src/containers/App/App.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { func } from 'prop-types';
 import { translate } from 'react-i18next';
-import 'i18n/i18n';
+import '../../i18n/i18n';
```

Comment:
Is there a way to configure storybook to resolve the `src` folder? Adding those extra `../../` will be painful when adding more components later on